lua/modules/MangaShiro.lua

```diff
@@ -21,7 +21,12 @@ end
 
 function getTitle(x)
 	local title = ''
-	if MODULE.ID == 'b53534f8443e420ea088594c53a3ff39' then title = x.XPathString('//div[@class="thumb"]/img/@alt') end
+
+	local titles = {
+		['b53534f8443e420ea088594c53a3ff39'] = '//div[@class="thumb"]/img/@alt', -- Manhwaland
+		['b5512eeeebbe4aa1a9194f58e8401ca2'] = '//div[@class="thumb"]/img/@alt', -- KumaScans
+	}
+	if titles[MODULE.ID] ~= nil then title = x.XPathString(titles[MODULE.ID]) end
```

Comment:
Why is this needed?
The title still show up properly even without this.

Reply:
It resolve a issue where some terms other than the title are also displayed, like the language or the format.
For exemple, the title of this [manga](https://kumascans.com/manga/true-education/) is True Education, but it is displayed as True Education English.
I think the solution with gsub is prone to some errors, like if the title really have the term "english" at the end. For  Manhwaland, most titles are prefixed with "Manwha".

Reply:
I see. In that case maybe do it like this `:gsub('English$', '')`?
That way it'll only remove it if the word is in the last title.

Reply:
It may work if a add this rule on a module basis, but at this point the alt of the img also has the title without this trick.

My concern is about all the gsub on line 40 and 41, those are generic and apply on all titles, not only on the one where the website add those terms.
Whap happens if one day a manga have a title with one of those terms? The title will be cut and that will be an issue.

Just for the sake of an exemple, if one of the website in this module host the manga 'I can transform!', the title will be transformed to "I  transform!".
If I add this gsub not based on the module ID and one day a manga with the title "I Teach English" is created and hosted, the title will be shortened.

Reply:
Then just add `if title == '' then title = x.XPathString('//div[@class="thumb"]/img/@alt') end` at the top. I think it should be fine. When the website has that string, it should only contains the proper title, right?

Yeah, while you at it you can remove `:gsub('can', '')` safely because the website `Mangacan` doesn't have that word in the title anymore.

Reply:
I have made these modifications.